mobileconnect-example/src/main/java/com/telenor/connect/mobileconnectexample/ExampleApplication.java

```diff
@@ -0,0 +1,23 @@
+package com.telenor.connect.mobileconnectexample;
+
+import android.app.Application;
+
+import com.telenor.connect.ConnectSdk;
+import com.telenor.mobileconnect.operatordiscovery.OperatorDiscoveryConfig;
+
+public class ExampleApplication extends Application {
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        ConnectSdk.sdkInitializeMobileConnect(
+                getApplicationContext(),
+                OperatorDiscoveryConfig
+                        .builder()
+                        .endpoint("https://discover.mobileconnect.io/gsma/v2/discovery")
+                        .clientId("abce79a0-5a73-488e-9389-4bc7f2685b5b")
+                        .clientSecret("923901ca-6977-4f90-b18a-34da0c5c7b69")
```

Comment:
It is documented, there is a disclaimer in README.md. 

The problem is in the nature of Operator Discovery. We (the MC team) have discussed this issue during our standups, and agreed on making a simplest possible example which "hides" its secrets in the compiled code, simultaneously encouraging developers to implement their own secret hiding scheme. The fact that we don't know exactly how they are supposed to do that, it is another issue. That's why Operator Discover sucks. The architecture of it is a political decision, rather than a technical one :(